OcarinaSongs.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import annotations
 import random
 import sys
-from collections.abc import Callable, Sequence
+from collections.abc import Sequence
```

Comment:
Upon re-reviewing, I noticed that you removed this import which is used in type aliases in lines 19 and 20. We should probably run a type checker as part of CI to catch errors like this. There may be other similar ones in this PR, haven't gone through everything yet.